core/src/main/java/org/jobrunr/jobs/context/JobRunrDashboardLogger.java

```diff
@@ -10,7 +10,7 @@
 
 public class JobRunrDashboardLogger implements Logger {
 
-    private static final ThreadLocal<JobDashboardLogger> jobDashboardLoggerThreadLocal = new ThreadLocal<>();
+    private static JobDashboardLogger jobDashboardLogger;
```

Comment:
this change is actually unrelated, I just had it on my branch and would be happy to revert and submit in a separate PR.
I noticed that `JobRunrDashboardLogger` was only working if called from the same thread where the job was started. Calling from a different thread would not log anything, this change should fix it

Reply:
Reverted